modules/machine-health-checks-about.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * master/deploying-machine-health-checks.adoc
+
+[id='machine-health-checks-about-{context}']
+= About `MachineHealthChecks`
+
+`MachineHealthChecks` are a specialized implementation of the `MachineAutoscaler`
+that automatically repairs unhealthy `Machines` in a particular `MachinePool`.
+
+To monitor machine health, you create a resource to define the
+configuration for a controller. You set a condition to check for, such as
+staying in the `NotReady` status for 15 minutes or displaying a permanent condition
+in the node-problem-detector, and a label for the set of machines to monitor.
+
+[NOTE]
+====
+You cannot apply a `MachineHealthCheck` to a machine with the master role.
+====
+
+The controller that observes a `MachineHealthCheck` resource checks for the status
+that you defined. If a machine fails the health check, it is automatically deleted
+and a new one is created to take its place. When a machine is deleted, you
+see <this kind of > event. The controller uses the same logic as the
+link:https://kubernetes.io/docs/concepts/architecture/nodes/#node-controller[Kubernetes node controller]
+to limit disruptive impact of the machine deletion.
+
+
+To stop the check, you remove the resource.
```

Comment:
Is there any more conceptual information that a user might need to know to deploy a machine health check?

Reply:
Just what is described in the docs we have: https://github.com/openshift/machine-api-operator#machine-healthcheck-controller